addons/packages/antrea/1.5.2/package.yaml

```diff
@@ -0,0 +1,279 @@
+                    portRange:
+                      type: string
+                      description: Provide the port range used by NodePortLocal.
+                      default: ""
+                antreaProxy:
+                  type: object
+                  additionalProperties: false
+                  description: AntreaProxy related configuration options.
+                  properties:
+                    proxyAll:
+                      type: boolean
+                      description: ProxyAll tells antrea-agent to proxy all Service traffic.
+                      default: false
+                    nodePortAddresses:
+                      type: array
+                      description: Specifies the host IPv4/IPv6 addresses for NodePort.
+                      items:
+                        type: string
+                        default: ""
+                      default: []
+                    skipServices:
+                      type: array
+                      description: List of Services which should be ignored by AntreaProxy.
+                      items:
+                        type: string
+                        default: ""
+                      default: []
+                    proxyLoadBalancerIPs:
+                      type: boolean
+                      description: Load-balance traffic destined to the External IPs of LoadBalancer services.
+                      default: false
+                kubeAPIServerOverride:
+                  type: string
+                  nullable: true
+                  description: Provide the address of Kubernetes apiserver.
+                  default: null
+                transportInterface:
+                  type: string
+                  nullable: true
+                  description: The name of the interface on Node which is used for tunneling or routing the traffic.
+                  default: null
+                transportInterfaceCIDRs:
+                  type: array
+                  description: The network CIDRs of the interface on Node which is used for tunneling or routing the traffic.
+                  items:
+                    type: string
+                    default: ""
+                  default: []
+                multicastInterfaces:
+                  type: array
+                  description: The names of the interfaces on Nodes that are used to forward multicast traffic.
+                  items:
+                    type: string
+                    default: ""
+                  default: []
+                tunnelType:
+                  type: string
+                  description: Tunnel protocols used for encapsulating traffic across Nodes. One of the following options => geneve, vxlan, gre, stt
+                  default: none
+                trafficEncryptionMode:
+                  type: string
+                  description: Determines how tunnel traffic is encrypted. One of the following options => none, ipsec, wireGuard
```

Comment:
Same as addons/packages/antrea/1.5.2/bundle/config/schema.yaml line 65.

Reply:
only a description, agreed to fix on another PR